src/services/rfqm_service.ts

```diff
@@ -92,6 +97,16 @@ export interface MetaTransactionRfqmQuoteResponse extends BaseRfqmQuoteResponse
     orderHash: string;
 }
 
+export interface SubmissionsMapStatus {
+    isTxMined: boolean;
+    isTxFinalized: boolean;
+    submissionsMap: SubmissionsMap;
+}
+
+export interface SubmissionsMap {
```

Comment:
Two code style points to consider:
I would avoid "Map" in the name; a reader can see the type. `TransactionHashtoSubmissionRecord` or something would be more descriptive.
Maybe use a [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map)